src/actions/emojis.js

```diff
@@ -53,7 +54,7 @@ export function getAllCustomEmojis(perPage = General.PAGE_SIZE_MAXIMUM) {
                     emojis = await Client.getCustomEmojis();
                     hasMore = false;
                 } else {
-                    emojis = await Client4.getCustomEmojis(page, perPage);
+                    emojis = await Client4.getCustomEmojis(page, perPage, 'name');
```

Comment:
what does 'name' mean?

Reply:
Sorting by name, I can make it a constant if that would make it clearer